sycl/doc/extensions/supported/sycl_ext_intel_queue_index.asciidoc

```diff
@@ -43,11 +43,7 @@ SYCL specification refer to that revision.
 
 == Status
 
-This is a proposed extension specification, intended to gather community
-feedback.  Interfaces defined in this specification may not be implemented yet
-or may be in a preliminary state.  The specification itself may also change in
-incompatible ways before it is finalized.  *Shipping software products should
-not rely on APIs defined in this specification.*
+This extension is implemented and fully supported by DPC++.
```

Comment:
The spec says:
> The `queue` constructor throws an `exception` with `errc::invalid` if
the index is less than `0`

If negative numbers do not have good semantics, I think we should just define the "index" type as unsigned.

Reply:
The spec makes it an "int" and that type is user-visbile, so it was easier to pass it like that between all the layers. How would having an "unsigned" help us?

Reply:
Remove this sentence from the spec and not have to check/deal with negative numbers. This is the first implementation so we can change the spec without problems, I think. 

Tag @gmlueck

Reply:
we decided offline to go with current spec